src/platforms/node/index.mdx

```diff
@@ -36,12 +36,52 @@ import * as Sentry from "@sentry/node";
 Sentry.init({ dsn: "___PUBLIC_DSN___" });
 ```
 
-## Usage
+## Verify Setup
+
+Verify your setup by intentionally sending an error that breaks your application. For example, in JavaScript, calling an undefined function throws an exception:
 
 ```javascript
 try {
-  // ...
+  myUndefinedFunction();
 } catch (e) {
   Sentry.captureException(e);
 }
 ```
+
+Resolve the created error by logging in to Sentry, then opening your project. On the **Issue Details** page, scroll down to the Exception stack trace and view the error.
+
+## Monitor Performance
+
+Performance Monitoring helps you see everything from macro-level metrics to micro-level spans, and you’ll be able to cross-reference transactions with related issues, customize queries based on your personal needs, and substantially more.
+
+To get started with performance monitoring with Node.js, first install these packages:
+
+```bash
+# Using yarn
+$ yarn add @sentry/node @sentry/tracing
+
+# Using npm
+$ npm install @sentry/node @sentry/tracing
+```
+
+To send traces, set the `tracesSampleRate` to a nonzero value. The following configuration will capture 100% of all your transactions:
+
+```javascript
+const Sentry = require("@sentry/node");
+// or use es6 import statements
+// import * as Sentry from '@sentry/node';
+
+// This is required since it patches functions on the hub
+const Tracing = require("@sentry/tracing");
+// or use es6 import statements
+// import * as Tracing from '@sentry/tracing';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampleRate: 1.0, // Be sure to lower this in production
+});
+```
+
+Performance data is transmitted using a new event type called "transactions", which you can learn about in [Distributed Tracing](/performance-monitoring/distributed-tracing/#traces-transactions-and-spans). **To capture transactions, you must install the performance package and configure your SDK to set the `tracesSampleRate` option to a nonzero value.** The example configuration above will transmit 100% of captured transactions. Be sure to lower this value in production otherwise you could burn through your quota quickly.
```

Comment:
```suggestion
Performance data is transmitted using a new event type called "transactions", which you can learn about in [Distributed Tracing](/performance-monitoring/distributed-tracing/#traces-transactions-and-spans). **To capture transactions, you must install the performance package and configure your SDK to set the `tracesSampleRate` option to a nonzero value.** The example configuration above will transmit 100% of captured transactions. You will want to lower this value in production, otherwise you could burn through your quota quickly.
```